### ISSUE TITLE
Overfitting except if you run it only in Ravdess_Music

### ISSUE THREAD

**Issue by hellfireworld** [marcogdepinto/Emotion-Classification-Ravdess#12]
Very good work but I managed to obtain almost the same accuracy and loss only if I run the code in Ravdess_Music without Ravdess_Speech !  Else If I run it in both I get overfitting. The model has very good accuracy and loss and poor val_accuracy and val_loss

**Comment by marcogdepinto** [comment]
Hi, this model is built only to solve this problem, predicting emotions on the RAVDESS dataset. Overfitting is correct in this case because is what I wanted to achieve. If you want a more generalized model, feel free to reduce the number of layers in the neural network and/or remove/add different training data. This is not an issue but expected behaviour, hence I am resolving the issue.
